src/picosvg/svg.py

```diff
@@ -280,13 +280,13 @@ def _resolve_clip_path(self, clip_path_url):
         clip_path = svg_pathops.union(*[from_element(e) for e in clip_path_el])
         return clip_path
 
-    def _combine_clip_paths(self, clip_paths):
+    def _combine_clip_paths(self, clip_paths) -> SVGPath:
         # multiple clip paths leave behind their intersection
-        if len(clip_paths) > 1:
-            return svg_pathops.intersection(*clip_paths)
-        elif clip_paths:
-            return clip_paths[0]
-        return None
+        if not clip_paths:
+            raise ValueError("Cannot combine no clip_paths")
+        if len(clip_paths) == 1:
+            clip_paths[0]
```

Comment:
what's this statement doing? was it meant to return?

Reply:
...yes!